src/MainUpdate.elm

```diff
@@ -890,12 +890,26 @@ update msg model =
                 )
 
             AddNewInformation ->
-                ( { model
-                    | filesToSave =
-                        appendNewElementToList model.filesToSave (FileToSave 0 "" "" "")
-                  }
-                , Cmd.none
-                )
+                let
+                    highestSavedFileId =
+                        model.savedFiles
+                            |> List.map .id
+                            |> List.maximum
+                            |> Maybe.withDefault 0
+
+                    newfileToSaveWithId =
+                        FileToSave (highestSavedFileId + List.length model.filesToSave + 1)
```

Comment:
does the id need to be the highest plus length of filesToSave plus 1? Not just highest plus one?

Reply:
Yeah that's correct. Because sometimes there will be multiple filesToSave